config/config.h.in

```diff
@@ -5,6 +5,10 @@
 // SPDX-License-Identifier: LGPL-2.1-only
 //////////////////////////////////////////////////////////////////////////////
 
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
```

Comment:
Why is this needed?

Reply:
To be honest, I am no longer certain.  This was previously spread among multiple source files so I moved it here.  But, I'll check if it can be removed or document why it is there.

Reply:
I removed these declarations as they are not being used.